apps/test/integration/levelSolutions/applab/formatting_visual.js

```diff
@@ -0,0 +1,103 @@
+import {TestResults} from '@cdo/apps/constants';
+export default {
+  app: "applab",
+  skinId: "applab",
+  levelFile: "levels",
+  levelId: "ec_simple",
+  tests: [
+    {
+      description: "old-sized app is scaled to new size",
+      editCode: true,
+      xml: '',
+      startHtml: `
+          <div id="divApplab" class="appModern running" tabindex="1" style="width: 320px; height: 450px;">
+            <div class="screen" tabindex="1" id="screen1" style=
+            "display: block; height: 480px; width: 320px; left: 0px; top: 0px; position: absolute; z-index: 0;">
+            <div class="screen" tabindex="1" id="screen2" style=
+            "display: block; height: 480px; width: 320px; left: 0px; top: 0px; position: absolute; z-index: 0;">
+            </div>
+          </div>
+          <div id="designModeViz" class="appModern withCrosshair clip-content" data-radium="true"
+          style="width: 320px; height: 450px; display: block;">
+            <div class="screen" tabindex="1" id="design_screen2" style=
+            "display: block; height: 480px; width: 320px; left: 0px; top: 0px; position: absolute; z-index: 0;">
+            </div>
+          </div>`,
+      runBeforeClick: function (assert) {
+        //Correct transform applied
+        assert.equal($('#phoneFrameWrapper').css('transform'), 'matrix(0.9375, 0, 0, 0.9375, 0, 0)');
+
+        //Screens appear at correct height
+        var screens = document.getElementsByClassName('screen');
```

Comment:
nit: We've been trying to avoid `var` in new code, instead preferring the ES6 syntax of `const` or `let`. In general, you should use `const` where you can and `let` if it's a variable that we mutate (such as i/j in your for loops)